File: backend/src/main/java/by/fpmibsu/findafriend/controller/commands/adverts/CreateAdvertCommand.java
package by.fpmibsu.findafriend.controller.commands.adverts;

import by.fpmibsu.findafriend.application.mediatr.Request;

public class CreateAdvertCommand extends Request<Boolean> {
    public int advertType;
    public String title;
    public String description;
    public int placeId;
}

File: BackendNET/Backend/Commands/Adverts/CreateAdvert/CreateAdvertCommand.cs
using System.Text.Json.Serialization;
using Backend.Entities;
using MediatR;

namespace Backend.Commands.Adverts.CreateAdvert;

public class CreateAdvertCommand : IRequest<bool>
{
    public Advert.AdvertType AdvertType { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public int PlaceId { get; set; }
    [JsonIgnore]
    public int UserId { get; set; }
}